api/client/output.proto

```diff
@@ -13,17 +14,39 @@ message Counter {
 }
 
 message Percentile {
-  google.protobuf.Duration duration = 1;
+  oneof duration_type {
+    google.protobuf.Duration duration = 1;
+    double raw_value = 4;
+  }
   double percentile = 2;
   uint64 count = 3;
 }
 
 message Statistic {
+  enum StatisticDomain {
+    RAW = 0;      // When this is set, the raw_* uint64 valued fields will be populated.
+    DURATION = 1; // When this is set, the duration-valued fields will be populated.
+  }
   uint64 count = 1;
   string id = 2;
-  google.protobuf.Duration mean = 3;
-  google.protobuf.Duration pstdev = 4;
+  oneof mean_type {
+    google.protobuf.Duration mean = 3;
+    double raw_mean = 8;
```

Comment:
Not sure if "raw_" is the best prefix / name. Can we try to document what is the meaning of raw in each one of these cases and then we will see? Is it always bytes? Something else?

Reply:
The definition of the `_raw` post-fixed fields map 1:1 to the return types of the corresponding getters from our internal `Statistics` c++ interface. In this PR we will use them to track the number of header bytes and response bytes, but possibly we may want to re-use these in the future to track things that aren't a "number of bytes"? (e.g. track number of requests over a single connection)